app/assets/stylesheets/dm/components/page_component.scss

```diff
@@ -152,6 +152,25 @@
     }
   }
 
+  .usa-img {
+    width: 100%;
+    height: auto;
+    max-height: 500px;
```

Comment:
set a max height as smaller pics were stretching out to fill the width and taking up what seemed like too much screen space compared to it's adjacent text column

Reply:
@PhilipDeFraties don't kill me but... I found out there are utility classes for this too 😂 https://designsystem.digital.gov/utilities/height-and-width/

Reply:
@camillevilla do you mean for setting the max height or the class in general? From what I was seeing the uswds does not have image resizing capability so there is nothing to replace `object-fit: cover`